### PR TITLE
Adding S3 Links For SSM_Bucket_Name Param

### PR DESCRIPTION
**Pull request by prat98**

### CODE REVIEW

File: stories/topics/con-aws-update-backup-data-file.adoc [hunk 1]
```diff
@@ -26,12 +26,12 @@ Please ensure the referenced backup vault exists and the AWS Backup IAM role pro
 * `aws_s3_bucket` is the name of the S3 bucket where the backup files are stored. 
 This creates a new bucket if one does not already exist. 
 Every backup is stored in the bucket. 
-For guidance on bucket naming, see link:https://www.google.com/url?q=https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html&sa=D&source=docs&ust=1682584492105680&usg=AOvVaw0WgUwkieoKwP4tRrvSpFBC[Bucket naming rules]
+For guidance on bucket naming, see link:https://www.google.com/url?q=https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html&sa=D&source=docs&ust=1682584492105680&usg=AOvVaw0WgUwkieoKwP4tRrvSpFBC[Bucket naming rules]. For more information on creating S3 buckets, see link:https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html[AWS Creating A Bucket] in the AWS documentation.
```
Comment (ianf77) [inline stories/topics/con-aws-update-backup-data-file.adoc:29]: can you put a line break between the two cross references?

**Review by ianf77 — COMMENTED** [review]
One comment.

File: stories/topics/con-aws-update-backup-data-file.adoc [hunk 2]
```diff
@@ -25,8 +25,9 @@ Please ensure the referenced backup vault exists and the AWS Backup IAM role pro
 * `aws_region` is the region where the foundation deployment is deployed.
 * `aws_s3_bucket` is the name of the S3 bucket where the backup files are stored. 
 This creates a new bucket if one does not already exist. 
-Every backup is stored in the bucket. 
-For guidance on bucket naming, see link:https://www.google.com/url?q=https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html&sa=D&source=docs&ust=1682584492105680&usg=AOvVaw0WgUwkieoKwP4tRrvSpFBC[Bucket naming rules]. For more information on creating S3 buckets, see link:https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html[AWS Creating A Bucket] in the AWS documentation.
+Every backup is stored in the bucket. +
```
Comment (ianf77) [inline stories/topics/con-aws-update-backup-data-file.adoc:28]: Remove the + at the end of the line

File: stories/topics/con-aws-update-backup-data-file.adoc [hunk 3]
```diff
@@ -25,8 +25,9 @@ Please ensure the referenced backup vault exists and the AWS Backup IAM role pro
...
+For guidance on bucket naming, see link:https://www.google.com/url?q=https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html&sa=D&source=docs&ust=1682584492105680&usg=AOvVaw0WgUwkieoKwP4tRrvSpFBC[Bucket naming rules]. +
```
Comment (ianf77) [inline stories/topics/con-aws-update-backup-data-file.adoc:29]: Remove the + at the end of the line.

File: stories/topics/con-aws-update-backup-data-file.adoc [hunk 4]
```diff
@@ -25,8 +25,9 @@ Please ensure the referenced backup vault exists and the AWS Backup IAM role pro
...
+For guidance on creating S3 buckets, see link:https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html[AWS Creating a bucket].
```
Comment (ianf77) [inline stories/topics/con-aws-update-backup-data-file.adoc:30]: put a blank line between these two xrefs

Comment (prat98) [reply stories/topics/con-aws-update-backup-data-file.adoc:30]: I added the `+` to create a line break. If I add a blank line and remove the +, things look off again.

It looks like this when removing the + and adding a blank line
Screenshot 2023-07-27 at 2 53 09 PM



I wanted it to look like below
Screenshot 2023-07-27 at 2 49 28 PM
ebf-bb4786ad8bd2">

**Pull request merged**